docs/spark-procedures.md

```diff
@@ -493,6 +493,39 @@ CALL spark_catalog.system.add_files(
 )
 ```
 
+### `register_table`
+
+Creates a catalog entry for a metadata.json file which already exists but does not have a corresponding catalog identifier.
+
+#### Usage
+
+| Argument Name | Required? | Type | Description |
+|---------------|-----------|------|-------------|
+| `table`       | ✔️  | string | Table which is to be registered |
+| `metadata_file`| ✔️  | string | Metadata file which is to be registered as a new catalog identifier |
+
+{{< hint warning >}}
```

Comment:
I'm not familiar with this syntax, I believe we do ```!!! Warning```

Like

```
!!! Warning
    Spark 3.0.0 has a correctness bug that affects dynamic `INSERT OVERWRITE` with hidden partitioning, [SPARK-32168][spark-32168].
    For tables with [hidden partitions](./partitioning.md), make sure you use Spark 3.0.1.
```

Reply:
Sure, I have made the change.